src/cole-slaw.md

```diff
@@ -0,0 +1,41 @@
+# Cole Slaw
+
+This is a cole slaw recipe that I got from a chili restaurant in my neighborhood that closed down a few years ago. This recipe makes more dressing than needed for the slaw, so you can save it or use it as a vegetable dip.
+
+- ⏲️ Prep time: 25 min
+- 🥶 Fridge time: 1 hr
+- 🍽️ Servings: 8 - 10
+
+## Ingredients
+
+- 3/4 mayonnaise
```

Comment:
3/4 of what? Is this 3/4 cup?

Reply:
This is common when referring to smaller, "child-size" versions of instruments. For example a child might start learning guitar on a 1/2 guitar, later move to a 3/4 guitar before his hands grow big enough for the regularly sized instrument.